tests/acceptance/conftest.py

```diff
@@ -103,182 +83,303 @@ def deployer_account(accounts):
+        proxy_contract.address, sender=deployer_account
     )
-    blockchain = taco_application_agent.blockchain
-    minimum_stake = (
-        taco_application_agent.contract.functions.minimumAuthorization().call()
+
+    return proxy_contract
+
+
+@pytest.fixture(scope="module")
+def coordinator(
+    nucypher_dependency, deployer_account, taco_child_application_proxy, ritual_token
+):
+    _coordinator = deployer_account.deploy(
+        nucypher_dependency.Coordinator,
+        taco_child_application_proxy.address,
+        TIMEOUT,
+        MAX_DKG_SIZE,
+        deployer_account.address,
+        ritual_token.address,
+        FEE_RATE,
+    )
+    _coordinator.makeInitiationPublic(sender=deployer_account)
+    taco_child_application_proxy.initialize(
+        _coordinator.address, sender=deployer_account
+    )
+    return _coordinator
+
+
+@pytest.fixture(scope="module")
+def global_allow_list(nucypher_dependency, deployer_account, coordinator):
+    contract = deployer_account.deploy(
+        nucypher_dependency.GlobalAllowList,
+        coordinator.address,
+        deployer_account.address,
+    )
+
+    return contract
+
+
+@pytest.fixture(scope="module")
+def subscription_manager(nucypher_dependency, deployer_account):
+    _subscription_manager = deployer_account.deploy(
+        nucypher_dependency.SubscriptionManager,
     )
+    return _subscription_manager
+
+
+#
+# Deployment/Blockchains
+#
+
+@pytest.fixture(scope='module', autouse=True)
+def deployed_contracts(
+    ritual_token,
+    t_token,
+    nu_token,
+    threshold_staking,
+    proxy_admin,
+    taco_application_proxy,
+    taco_child_application_proxy,
+    coordinator,
+    global_allow_list,
+    subscription_manager,
+):
+    # TODO: can this be improved - eg. get it from the project fixture
+    deployments = [
+        ritual_token,
+        t_token,
+        nu_token,
+        threshold_staking,
+        proxy_admin,
+        taco_application_proxy,  # only proxy contract
+        taco_child_application_proxy,  # only proxy contract
+        coordinator,
+        global_allow_list,
+        subscription_manager,
+    ]
+    return deployments
+
+
+@pytest.fixture(scope='module', autouse=True)
+def test_registry(deployed_contracts):
+    registry = registry_from_ape_deployments(deployments=deployed_contracts)
+    return registry
+
+
+@pytest.fixture(scope='module')
+def testerchain(project, test_registry) -> TesterBlockchain:
+    # Extract the web3 provider containing EthereumTester from the ape project's chain manager
+    provider = project.chain_manager.provider.web3.provider
+    testerchain = TesterBlockchain(eth_provider=provider)
+    BlockchainInterfaceFactory.register_interface(interface=testerchain, force=True)
+    yield testerchain
+
+
+#
+# Staking
+#
+@pytest.fixture(scope="module")
+def staking_providers(
+    deployer_account,
+    accounts,
+    testerchain,
+    test_registry,
+    threshold_staking,
+    taco_application_proxy,
+):
```

Comment:
✔️ - Whenever you see this it's either a fixture dependency or unneeded parameter. In this case it was the former. However, that can be address by using `@pytest.mark.usefixtures("<name of dependent fixture>").

We still need that fixture for set up, but not as a parameter since it isn't directly used.